### DISCUSSION TITLE
posibilidad de eliminar taller 3.

### DISCUSSION THREAD

**Discussion by sebasvelasco353** [sebasvelasco353/FDP-2022_01#162] [Announcements]
Hola guys como van?
como les comentaba en la clase de hoy, todavia nos falta un taller antes del final, ese taller vale el 20% de la materia, al igual que los otros dos, admeas de eso tenemos seguimientos (ejercicios de los viernes) que valen 10% y el final que vale 30%.

les propongo que eliminemos ese taller 3 y distribuimos su valor de la siguiente forma:

Taller 1 -> 25%
Taller 2 -> 25%
Ejercicios en clase -> 20%
final -> 30%

porfa si les parece buena idea comenten en este anuncio confirmando.

Gracias.

**Comment by Darksea1220** [comment]
confirmando

**Reply by AnnaHenao131** [reply]
Confirmado

**Comment by Karolfc** [comment]
Estoy de acuerdo :D

**Comment by DParuma** [comment]
Me parece bien 😸

**Comment by Axelito02** [comment]
Confirmo

**Comment by salazar2356** [comment]
Confirmao

**Comment by Juli0931** [comment]
Confirmandoando

**Comment by sebasvelasco353** [comment]
Hola guys, como lo mencione la clase pasada vamos a hacer un tercer taller, esto debido a que en el taller 2 nadie uso clases y me veo obligado a calificarles el uso de clases y objetos de esta forma.

dejo esto como constancia de lo hablado la clase anterior y las notas de la materia quedan de la siguiente forma

taller 1 -> 20%
taller 2 -> 20%
taller 3 -> 20%
final en grupos con sustentacion -> 30%
ejercicios en clase de los viernes -> 10%

nos vemos en clase.
